config/schedule.rb

```diff
@@ -0,0 +1,3 @@
+every :day, at: '11:00 am' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.